### PR TITLE
Change example for orderby to use ASC rather than DESC

### PR DESCRIPTION
**Pull request by mkindahl**
# Description

[Short summary of why you created this PR]

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: use-timescale/compression/about-compression.md [hunk 1]
```diff
@@ -154,13 +154,13 @@ showing numbers around 88, and device 2 showing numbers around 300:
 |-|-|-|-|-|
 |[12:00:02, 12:00:02, 12:00:01, 12:00:01 ]|[1, 2, 1, 2]|[88.2, 300.5, 88.6, 299.1]|[20, 30, 25, 40]|[0.8, 0.9, 0.85, 0.95]|
 
-To improve the performance of this data, you can order by `device_id, time DESC`
+To improve the performance of this data, you can order by `device_id, time ASC`
```
Comment (svenklemm) [inline use-timescale/compression/about-compression.md]:
```suggestion
To improve the performance of this data, you can order by `device_id, time`
```

File: use-timescale/compression/about-compression.md [hunk 2]
```diff
@@ -154,13 +154,13 @@ showing numbers around 88, and device 2 showing numbers around 300:
...
 instead, using these commands:
 
 ```sql
 ALTER TABLE  example
   SET (timescaledb.compress,
-       timescaledb.compress_orderby = 'device_id, time DESC');
+       timescaledb.compress_orderby = 'device_id, time ASC');
```
Comment (svenklemm) [inline use-timescale/compression/about-compression.md]:
```suggestion
       timescaledb.compress_orderby = 'device_id, time');
```

**Review by svenklemm — APPROVED** [review]
Suggest to remove ASC alltogether since it is the default and non-default sort options are a subtle footgun

File: use-timescale/compression/about-compression.md (hunk 1 above)
Comment (mkindahl) [reply use-timescale/compression/about-compression.md:157]: I do not think that `ASC` is the default (yet?) and the text should also provide good guidance for users before timescale/timescaledb#6299?

**Comment by mkindahl** [comment]
> Suggest to remove ASC alltogether since it is the default and non-default sort options are a subtle footgun

I do not think that `ASC` is the default (yet?, but it is in timescale/timescaledb#6299) and the text should also provide good guidance for users before timescale/timescaledb#6299?

**Comment by svenklemm** [comment]
ASC is the default if you do not specify ASC/DESC, the default for no orderby at all is `time DESC` but if you specify just `time` it will use standard ORDER BY parsing rule and be equivalent to `time ASC`

**Comment by billy-the-fish** [comment]
@svenklemm, @mkindahl , am I ok to merge and publish or shall I close this issue?

**Comment by svenklemm** [comment]
No objections to merging

**Pull request merged**